PS8lite.Rmd

```diff
@@ -184,12 +185,38 @@ MiscVal + MoSold + YrSold" %>%
 
 # Fit a Random Forest model
 
-Fit one Random Forest model `model_rf` using any approach you want.
+Note that `caret::train` was used to find optimal `mtry`, which turned out to be 19. Process was commented out because it takes a bit of time to run/knit.
```

Comment:
This is a great example of a problem where computational constraints become an issue! In other words, method A might yield "better predictions" than method B, but if method A takes a week to run and method B takes only a few minutes, you might sacrifice a little predictive power for speed!